terasoluna-gfw-common-libraries/pom.xml

```diff
@@ -175,5 +175,17 @@
         <maven.javadoc.skip>true</maven.javadoc.skip>
       </properties>
     </profile>
+    <profile>
+      <id>jdk17</id>
+      <activation>
+        <jdk>17</jdk>
+      </activation>
+      <properties>
+        <!-- coverall 4.3.0 or previous version does not work on Java 11, see https://github.com/trautonen/coveralls-maven-plugin/issues/112 -->
+        <coveralls.skip>true</coveralls.skip>
+        <!-- javadoc does not work on Java 11.0.2, see https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=920020 -->
```

Comment:
- [ ] The comments seem to be wrong here too.
By the way, have the bugs mentioned in the comments been resolved yet?
Well, perhaps it would be better to make this a separate topic as well.

Reply:
This problem seems to have already been resolved, and I tried running the plugin with java 11.0.13 and 17.0.3 and got a warning but no error.
Created an issue about the study of the setting.
https://github.com/terasolunaorg/terasoluna-gfw/issues/1180